source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -92,9 +92,9 @@ - (void)update:(NSString *) jsonStr
 {
     self.editableStr = jsonStr;
     ACRRenderResult *renderResult;
-    ACOParseResult *hostconfigParseResult = [ACOHostConfig fromJson:self.hostconfig];
-    ACOParseResult *cardParseResult       = [ACOAdaptiveCard fromJson:jsonStr];
-    if(hostconfigParseResult.IsValid && cardParseResult.IsValid)
+    ACOHostConfigParseResult *hostconfigParseResult = [ACOHostConfig fromJson:self.hostconfig];
+    ACOAdaptiveCardParseResult *cardParseResult       = [ACOAdaptiveCard fromJson:jsonStr];
+    if(cardParseResult.IsValid)
```

Comment:
>cardParseResult.IsValid [](start = 7, length = 23)

Not checking for hostconfig anymore?

Reply:
good point; it's up to users. i'm showing a scenario when user (I) accepts default host config when fails.